docs/bugs.json

```diff
@@ -1,4 +1,12 @@
 [
+    {
+        "name": "ArraySliceDynamicallyEncodedBaseType",
+        "summary": "Index range accesses on arrays with dynamically encoded base types can result in invalid array accesses.",
+        "description": "For arrays with dynamically sized base types, index range accesses that use a start expression that is non-zero will result in invalid array slices. Any index access to such array slices will result in data being read from incorrect calldata offsets.",
+        "introduced": "0.6.0",
+        "fixed": "0.6.8",
+        "severity": "very low"
+    },
```

Comment:
Should be abiencoderv2-only: v1 does not support dynamic arrays of dynamically encoded data, does it?

Reply:
Ah, in external functions it does!

Reply:
Does it? I think you're right and it *should* be abiencoderv2 only, let me check!

Reply:
This is *only* about dynamic calldata variables since otherwise we don't have slice support at all (maybe I should add that in the description as well). So they have to be external function arguments. ``typeSupportedByOldABIEncoder`` will always run on those and always return false as far as I see - I'll make it abiencoderv2-only, let me know if there's an exception I'm missing.